docs/concepts/gas.md

```diff
@@ -0,0 +1,200 @@
+---
+id: gas
+title: Gas
+sidebar_label: Gas
+---
+
+Gas is a dynamic, unified unit of cost of computation and storage on the blockchain as well as network utilization among nodes.  
```

Comment:
Better to remove "storage" from the sentence, since in the implementation the sender buys gas to pay for computation. Storage rent price is in tokens, since it has to be static all the time.

Reply:
oh, nice.  thank you for this clarification

Reply:
one small thing: saying "Gas is a dynamic" is confusing. Yes, gas amount per operation could change through the governance process, but in general - each operation has a fixed price in gas. Gas price is dynamic, but not gas itself.

Reply:
will rewrite to this.  is it better?

> Gas is a unified unit of cost of computation on the blockchain.  It is purchased at a variable price point (depending on system load) using NEAR tokens at the moment a transaction is processed.